.github/workflows/check-packetbeat.yml

```diff
@@ -0,0 +1,30 @@
+name: check-packetbeat
+
+on:
+  pull_request:
+    paths:
+      - '.github/workflows/check-packetbeat.yml'
+      - 'packetbeat/**'
+      - 'x-pack/packetbeat/**'
```

Comment:
Yes, the current implementation in Jenkins runs always, see:
- https://github.com/elastic/beats/blob/947c837f5e86210814863f9ffaa4bd8751dd0d9f/packetbeat/Jenkinsfile.yml#L17-L24

Originally implemented in https://github.com/elastic/beats/pull/23378 and https://github.com/elastic/beats/pull/23695

Reply:
OK. Is there a reason that there are two checks then? I must be missing something.

Reply:
There are two checks since there are two beats components (`packetbeat` and `x-pack/packetbeat`), each action runs a different set of steps.

* `packebeat` uses `make -C packetbeat check update; make check-no-changes`. 
* `x-pack/packetbeat` uses `cd ... ; mage check update`

In addition, the github check runs in both cases if any changes in those beats components. I could eventually create one monolithically github action, if so, there is a chance the github check fail fast and won't provide an holistic overview of those checks.

Reply:
OK. Thanks for clarifying.